libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.spec.ts

```diff
@@ -466,7 +484,12 @@ describe('Toggle switch component', () => {
       expect(formControl.dirty).toEqual(true);
       expect(formControl.value).toEqual(true);
 
-      SkyAppTestUtility.fireDomEvent(buttonElement, 'blur');
+      if (buttonElement) {
+        SkyAppTestUtility.fireDomEvent(buttonElement, 'blur');
+      } else {
+        fail('Expected button element to exist');
```

Comment:
?

Reply:
I updated `fireDomEvent` to handle `null` or `undefined` and now have removed the if statement